sbc_compassion/migrations/12.0.1.0.4/post-migration.py

```diff
@@ -0,0 +1,48 @@
+from openupgradelib import openupgrade
+
+
+def _update_reading_language(cr, sponsorship_id, lang):
+    cr.execute(
+        "UPDATE recurring_contract "
+        "SET reading_language = %s "
+        "WHERE id = %s" % (lang, sponsorship_id)
```

Comment:
The execute method already takes parameters for string replacement and should be used to avoid possible sql injections.
```suggestion
        "WHERE id = %s", [lang, sponsorship_id]
```